data/sql/updates/pending_db_world/rev_1643745379505594100.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643745379505594100');
+DELETE FROM `creature` WHERE  `guid` = 247680; /* Duplicate of 247673 */
```

Comment:
i guess move the notes above or remove them. code style change must of happened at some point i am not aware of.

Reply:
I think it is useful to know the duplicate IDs so if we can keep them it will be the best. I have moved them to above each line; hopefully this is enough for code style to be happy! Thank you for noticing this issue.

Reply:
Please kindly let me know if it does not pass and I will remove the comments completely. Thank you!